src/EditorFeatures/Core/Commands/AsyncFindReferencesCommandArgs.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Commands
+{
+    /// <summary>
+    /// Arguments for async find references
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    internal class AsyncFindReferencesCommandArgs : CommandArgs
```

Comment:
This is somewhat of a prototype.  So currently David and I are modeling this as a different command that comes in from VS and a different window that exists that presents the info.  The expectation would be that in the final form there would just be one command and one presenter.

We will have to do the right thing to keep working downlevel on Dev14 and light up on Dev15, but that doesn't seem like an issue.  And @Pilchie has down the work so that should be fine.